base/Config.h

```diff
@@ -24,8 +24,11 @@
 #endif
 
 #if EM_USE_SDL
-//#include <GL/gl.h>
-#include <SDL_opengl.h> //!+-rzr: better portability
+#ifdef HAVE_OPENGLES
+#include <GLES/gl.h>
```

Comment:
ok here but I think this should be done in SDL2, i'll try to open a PR there if you don't

```
/usr/include/SDL/SDL_opengl.h:#include <OpenGL/gl.h>    /* Header File For The OpenGL Library */
/usr/include/SDL/SDL_opengl.h:#include <OpenGL/glu.h>   /* Header File For The GLU Library */
/usr/include/SDL/SDL_opengl.h:#include <gl.h>           /* Header File For The OpenGL Library */
/usr/include/SDL/SDL_opengl.h:#include <glu.h>  /* Header File For The GLU Library */
/usr/include/SDL/SDL_opengl.h:#include <GL/gl.h>        /* Header File For The OpenGL Library */
/usr/include/SDL/SDL_opengl.h:#include <GL/glu.h>       /* Header File For The GLU Library */
```

Reply:
You're right, it's more consistent. Thanks.